tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_ConfiguredAssetSQLExample_SQL.ipynb

```diff
@@ -567,10 +557,53 @@
     "results.success"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "6737ccdd",
+   "metadata": {},
+   "source": [
+    "# Appendix\n"
+   ]
+  },
   {
    "cell_type": "markdown",
    "id": "6914725d-85cc-4c9d-a93d-2d1c329eac74",
    "metadata": {},
+   "source": [
+    "## Other Parameters for `ConfiguredAssetSqlDataConnector`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f8e62691",
+   "metadata": {},
+   "source": [
+    "The signature of the `ConfiguredAssetSqlDataConnector` also contains the following parameters: \n",
+    "\n",
```

Comment:
Appendix added

Reply:
I made some small suggestions to the appendix in https://github.com/great-expectations/great_expectations/pull/5958 that might be good to include in this notebook as well.